npc/analyser.go

```diff
@@ -50,6 +52,13 @@ func (ns *ns) analysePolicy(policy *networkingv1.NetworkPolicy) (
 			// If From is empty or missing, this rule matches all sources
 			allSources := ingressRule.From == nil || len(ingressRule.From) == 0
 
+			if !allPorts {
+				for _, npProtocolPort := range ingressRule.Ports {
+					if _, err := strconv.Atoi(port(npProtocolPort.Port)); err != nil {
+						return nil, nil, nil, nil, errors.New("named ports in network policies in not support yet. Rejecting network policy from further processing")
```

Comment:
"in not support" -> "is not supported"

Reply:
It's helpful to the end-user to print out the actual value that failed, e.g. `named port "http" in network ...`.  
Maybe create a function that wraps up the check and the error so you don't repeat it 60 lines later.